### PR TITLE
Bump min Android version to 16

### PR DESCRIPTION
**Pull request by guperrot**

### CODE REVIEW

File: Tests/Droid/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="53" android:versionName="1.9.1-SNAPSHOT" package="com.contoso.contoso_forms_test">
-    <uses-sdk android:minSdkVersion="15" />
-    <application android:label="Contoso.Forms.Test">
-    </application>
-</manifest>
+	<uses-sdk android:minSdkVersion="16" />
```
Comment (guperrot) [inline Tests/Droid/Properties/AndroidManifest.xml]: We modify via UI, formatting is weird but trying to reformat would be canceled by using the UI of Visual Studio.

File: Tests/Droid/Contoso.Forms.Test.Droid.csproj [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <AndroidManifest>Properties\AndroidManifest.xml</AndroidManifest>
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
-    <AndroidUseLatestPlatformSdk>true</AndroidUseLatestPlatformSdk>
+    <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
```
Comment (dhei) [inline Tests/Droid/Contoso.Forms.Test.Droid.csproj]: What is the reason of this change?

Comment (guperrot) [reply Tests/Droid/Contoso.Forms.Test.Droid.csproj]: To pin Mono compiler to 8.1 as we pinned mono version on VSTS

**Comment by TroubleMakerBen** [comment]
@MatkovIvan I updated the changelog and bumped the version of monoAndroid in the nuspecs.
